configs/arbitrum.goerli.json

```diff
@@ -15,7 +15,8 @@
     "liquidity": "0x42A460211d20EbCaA6982AE5f4b602dCB9973B1c"
   },
   "helpers": {
-    "trade": "0xEcE4e6457E68c88B11cEddA47B35942434Cf6e2a"
+    "trade": "0xEcE4e6457E68c88B11cEddA47B35942434Cf6e2a",
+    "maxPositionHelper": ""
```

Comment:
probably better naming than `maxPositionHelper` something like `limitHandlerExtension` or `limitHandlerHelper` so it is not too specific